mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -149,6 +149,8 @@ private[ml] trait HasVarianceCol extends Params {
    */
   final val varianceCol: Param[String] = new Param[String](this, "varianceCol", "Column name for the biased sample variance of prediction")
 
+  setDefault(varianceCol, "variance")
```

Comment:
You should not modify this file directly. Update `SharedParamsCodeGen.scala` instead.